tests/requirements_file_exercise_tests.py

```diff
@@ -0,0 +1,132 @@
+
+        mock_read_file_lines.assert_has_calls([call(main_requirements_file), call(referenced_requirement_file)])
+        self.assertEqual(excepted, actual)
+
+    @mock.patch('src.requirements_file_exercise.read_file_lines')
+    def test_create_libraries_string_should_add_just_one_instance_of_library_to_list(self, mock_read_file_lines):
+        mock_read_file_lines.return_value = ["Flask==2.0.2", "Flask-RESTful==0.3.9", "Jinja2==3.0.2",
+                                             "validators==0.18.2", "Flask==2.0.2", "Jinja2==3.0.2"]
+        file_name = 'path_to_rquirements_file.txt'
+        excepted = "Flask Flask-RESTful Jinja2 validators"
+
+        actual = create_libraries_string(file_name)
+
+        mock_read_file_lines.assert_called_once_with(file_name)
+        self.assertEqual(excepted, actual)
+
+    @mock.patch('src.requirements_file_exercise.read_file_lines')
+    def test_should_create_empty_string_for_empty_file(self, mock_read_file_lines):
+        excepted = ""
+        mock_read_file_lines.return_value = []
+        file_name = 'path_to_rquirements_file.txt'
+
+        actual = create_libraries_string(file_name)
+
+        mock_read_file_lines.assert_called_once_with(file_name)
+        self.assertEqual(excepted, actual)
+
+    @mock.patch('src.requirements_file_exercise.read_file_lines')
+    def test_should_read_once_file_that_has_more_than_one_reference(self, mock_read_file_lines):
+        first_requirements_file = 'requirements.txt'
+        second_requirements_file = 'second-requirements.txt'
+        third_requirements_file = 'third-requirements.txt'
+        mock_lines_of_first_requirement_file = ["Flask==2.0.2", "-r " + second_requirements_file, "Jinja2==3.0.2",
+                                                "-r " + third_requirements_file, "validators==0.18.2"]
+        mock_lines_of_second_requirement_file = ["requests==2.26.0", "-r " + third_requirements_file,
+                                                 "typing-extensions==3.10.0.2"]
+        mock_lines_of_third_requirement_file = ["six==1.16.0", "Flask-RESTful==0.3.9", "redis==2.10.6"]
+        mock_read_file_lines.side_effect = [mock_lines_of_first_requirement_file, mock_lines_of_second_requirement_file,
+                                            mock_lines_of_third_requirement_file]
+        excepted = "Flask Jinja2 validators requests typing-extensions six Flask-RESTful redis"
+
+        actual = create_libraries_string(first_requirements_file)
+
+        mock_read_file_lines.assert_has_calls([call(first_requirements_file), call(second_requirements_file),
+                                               call(third_requirements_file)])
+        self.assertEqual(mock_read_file_lines.call_count, 3)
+        self.assertEqual(excepted, actual)
+
+    @mock.patch('src.requirements_file_exercise.read_file_lines')
+    def test_should_create_libraries_string_that_referenced_file_has_reference_to_more_file(self, mock_read_file_lines):
```

Comment:
I didn't like the name

Reply:
By the way, you already check it in the test function above .... 
'def test_should_read_once_file_that_has_more_than_one_reference'

Even you have the same excepted:
excepted = "Flask Jinja2 validators requests typing-extensions six Flask-RESTful redis"